package.json

```diff
@@ -2,7 +2,7 @@
   "name": "twilio-video",
   "title": "Twilio Video",
   "description": "Twilio Video JavaScript Library",
-  "version": "2.19.0-dev",
+  "version": "2.19.1-dev",
```

Comment:
Curious why is this change showing?

Reply:
I had this branch created from `master` originally, and then pointed to develop. That must be it, But after merging develop into branch this change is not showing up anymore.